content/miq_dialogs/miq_provision_azure_dialogs_template.yaml

```diff
@@ -357,9 +357,23 @@
           :required: true
           :display: :edit
           :data_type: :string
+          :required_method: :validate_blacklist
+          :blacklist: [
+            'administrator', 'admin', 'user', 'user1', 'test', 'user2', 'test1', 'user3',
+            'admin1', '1', '123', 'a', 'actuser', 'adm', 'admin2', 'aspnet', 'backup',
+            'console', 'david', 'guest', 'john', 'owner', 'root', 'server', 'sql',
+            'support', 'support_388945a0', 'sys', 'test2', 'test3', 'user4', 'user5'
```

Comment:
Probably best to put them in alphabetical order I suppose. I can fix that. I'm not sure about "david" or "john", other than being super common first names, but "support_388945a0" is a default built-in user account. More info if you care:

https://technet.microsoft.com/en-us/library/cc779144(WS.10).aspx